package.json

```diff
@@ -0,0 +1,18 @@
+{
+	"name": "@types/mediawiki",
+	"version": "0.1.0",
+	"description": "TypeScript definitions of MediaWiki interfaces",
+	"scripts": {
+		"build:types": "tsc -p tsconfig.json",
+		"build": "npm-run-all build:*",
+		"test:lint": "eslint .",
+		"test": "npm-run-all test:*"
+	},
+	"license": "GPL-2.0-or-later",
+	"devDependencies": {
```

Comment:
Very, very minor (feel free to ignore): package lockfiles solve the problem of reproducible builds but contain a lot of noise. I think it would be much easier to track top-level upgrades if these packages were pinned to patch versions (omit the `^`). In this way, we deliberately upgrade by running something like `npm audit fix && npm ou && ncu -u && npm i` (where `ncu` is [npm-check-updates](https://www.npmjs.com/package/npm-check-updates)).